app/Main.hs

```diff
@@ -26,12 +28,35 @@ initScreen width height = do
         SDL.setWindowTitle window s
     return $ SDLContext window renderer
 
+getWindow :: SDLContext -> SDL.Window
+getWindow (SDLContext window _) = window
+
+getRenderer :: SDLContext -> SDL.Renderer
+getRenderer (SDLContext _ renderer) = renderer
+
+arial :: String
+arial = "../assets/arial.ttf"
+
 main :: IO ()
 main = do
     ctx <- initScreen 800 600
     ptrEvent <- malloc
-    draw ctx
-    eventLoop ptrEvent ctx
+    --draw ctx
+    
+    TTF.withInit $ do
+        font <- TTF.openFont arial 150
+        textSurface <- TTF.renderUTF8Solid font "test" (SDL.Color 255 255 255 0)
+        textTexture <- SDL.createTextureFromSurface (getRenderer ctx) textSurface
+        loop ptrEvent ctx textTexture 
```

Comment:
can you make a new constructor for `DisplayCommand` that takes a position, font name, font weight, color, and text, and basically encapsulates this logic? that way, the main application doesn't have to really know about the specifics of rendering, and it'll be easier to generate display text commands during the painting process